### PR TITLE
Testbranch

### PR DESCRIPTION
**Pull request by lmart296**
test

### CODE REVIEW

**Comment by lmart296** [comment]
hey this sucks

**Comment by ealva153** [comment]
garbage

File: Code/src/web/app.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /* jshint node: true */
 'use strict';
 
-var _               = require('lodash');
```
Comment (lmart296) [inline Code/src/web/app.js:4]: hey you shouldnt do this. easier to make it a var

Comment (pmess002) [inline Code/src/web/app.js:4]: Testing, yo.

**Pull request closed without merging**